src/lib/LBT/LBT.cpp

```diff
@@ -0,0 +1,153 @@
+#include "common.h"
+#include "logging.h"
+#include "LBT.h"
+
+extern SX1280Driver Radio;
+
+LQCALC<100> LBTSuccessCalc;
+volatile uint32_t rxStartTime;
+volatile bool LBTChannelBusy = false;
+
+#if !defined(LBT_RSSI_THRESHOLD_OFFSET_DB)
+  #define LBT_RSSI_THRESHOLD_OFFSET_DB 0
+#endif
+
+volatile bool LBTEnabled = false;
+volatile bool LBTScheduleDisable = false;
+volatile bool LBTStarted = false;
+
```

Comment:
* Remove LBTEnabled, LBTScheduleDisable, enableLBT() entirely. We only need to do LBT if >10mW right? So there's no need for turning it on and off externally based on configured power.
```
bool LbtIsEnabled()
{
  return POWERMGMT::currPower() > PWR_10mW;
}
```
Note that the implementation of currPower() should be moved the POWERMGMT.h so it can be inlined by the compiler, as the declaration in .cpp is not declared IRAM and inlining will move it into IRAM if needed if it is in the header.

* LBTStarted is not volatile, should be static.

Reply:
The idea with LBTenable() is to turn it on/off depending on lua-config of powerlevel. To not have to recompile in the field if the need for switching between more tx power or no LBT arises. Can this be solved without LBTenable()?

Reply:
Yes, because it is always on when compiled with the CE regdomain and the power is > 10mW right? So just check the current power level and it is on if the current power is > 10mW. The flags are redundant because they are set any time the config'd power changes and the config'd power is > 10mW, but at 10mW we don't need to check at all, so it can just look at currPower and decide per-packet.

Unless I am missing something?

Reply:
The way we understand the regulations is LBT also needs to remain enabled when using dynamic power and the power level drops down to 10mW.

Reply:
Ahh that's a bit weird haha you don't have to do it if the power is low enough, but if you're doing it, keep doing it even if the power goes low.

So we can leave LBTEnabled then, since there's not much overhead to that, but it does not need to be volatile and doesn't need a synchronized change does it? It is only changed when the radio is idle / switching configs.

Reply:
Agree, it doesn't need to be synchronized now, that was a remnant of some previous code I think...  fixed in latest commit

Reply:
Also moved currPower() to POWERMGMT.h as mentioned above here.